pkg/git/azdo.go

```diff
@@ -67,13 +68,14 @@ func NewAzdoGITProvider(ctx context.Context, remoteURL, token string) (*AzdoGITP
 }
 
 // CreatePR ...
-func (g *AzdoGITProvider) CreatePR(ctx context.Context, branchName string, auto bool, state *PRState) error {
+//nolint:funlen // temporary
```

Comment:
Temporary?

Reply:
Yes, other PRs will make changes that will take us back under the limit again.